content/en/docs/concepts/storage/volume-snapshots.md

```diff
@@ -98,16 +100,16 @@ spec:
 
 ### Class
 
-A VolumeSnapshotContent can have a class, which is specified by setting the
+A `VolumeSnapshotContent` can have a class, which is specified by setting the
 `snapshotClassName` attribute to the name of a
 [VolumeSnapshotClass](/docs/concepts/storage/volume-snapshot-classes/).
-A VolumeSnapshotContent of a particular class can only be bound to VolumeSnapshots requesting
-that class. A VolumeSnapshotContent with no `snapshotClassName` has no class and can only be bound
-to VolumeSnapshots that request no particular class.
+A `VolumeSnapshotContent` of a particular class can only be bound to *volume snapshots* requesting
```

Comment:
`VolumeSnapshots`?

Reply:
@xing-yang is this accurate? Doesn't `VolumeSnapshot` also need to explicitly specify the `VolumeSnapshotContentName`?

Reply:
Does the class only matter for dynamic provisioning?

Reply:
Content has the class name in the current API, but will be removed once the bringing snapshot to Beta API PR is merged.  The intent of this PR is not to address changes in the new Beta APIs.

Reply:
@xing-yang can you clarify my first question? Doesn't VolumeSnapshot need to request the exact VolumeSnapshotContent object? We only support explicit binding for snapshots, we don't pick from a pre-existing pool.